examples/messaging/messaging/client.py

```diff
@@ -1,36 +1,39 @@
 import threading
-import urllib2
+import requests
+from requests import Request, Session
+from requests.auth import HTTPBasicAuth
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from requests import RequestException
+
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```

Comment:
Actually if you've got warning it is because your installation of Python is not secure.
You should install `pip install PyOpenSSL cryptography idna  certifi`

Or simply `pip install urllib3[secure]`

See https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings